### PR TITLE
Tweak documentation and blog

### LINKED ISSUE #2843: Improve SEO for documentation and blog
### What should be done?

The documentation is not quite visible enough through search engines. We need to explore and implement ways to make all aspects of the documentation easily searchable through search engines.

### LINKED ISSUE #2847: Add guidelines for keywords and tags
### What should be done?

We don't currently have guidelines for tags and keywords for our blog.

- Keywords should be content-specific and SEO-related, so long tailed keyword are okay as long as it's related to content and helps search engine discovery. [Research current Docusaurus applications on this]
- Tags are case-sensitive in docusaurus, so a `FerretDB` and `ferretdb` are organized separately. View here https://blog.ferretdb.io/tags/
- All keywords and tags should be small-case
- All tags should be in the plural form, e.g. `document database` is best written as `document databases`. (This depends, as long as it's not present in the list of tags in whatever form)
- Hyphens should be disregarded, e.g. `mongodb-compatible database` should be written as `mongodb compatible database`
- List out all currently used tags, and any newly added tag should be added to the contribution guideline. Let's also establish a set of commonly used tags based on our content direction.

### PR DESCRIPTION
**Pull request by Fashander**
# Description

Closes #2843 
Closes #2847

Provided guidelines for tags and updated blog posts to match tags. Also updated documentation SEO.

## Readiness checklist

- [ ] I added/updated unit tests.
- [ ] I added/updated integration/compatibility tests.
- [ ] I added/updated comments and checked rendering.
- [ ] I made spot refactorings.
- [x] I updated user documentation.
- [ ] I ran `task all`, and it passed.
- [x] I ensured that PR title is good enough for the changelog.
- [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Labels, Project and project's Sprint fields.
- [x] I marked all done items in this checklist.

### CODE REVIEW

**Comment by AlekSi** [comment]
> Closes https://github.com/FerretDB/FerretDB/issues/2843 https://github.com/FerretDB/FerretDB/issues/2847

That will not work, it should be one "Closes #number" per line. "Conform PR" action will also fail.

File: website/docs/contributing/writing-guide.md [hunk 1]
```diff
@@ -77,6 +77,39 @@ Rather than use relative paths, we strongly suggest the following approach, sinc
 
 `![FerretDB logo](/img/logo-dark.png)`.
 
+## Tags
```
Comment (AlekSi) [inline website/docs/contributing/writing-guide.md:80]: Could we also briefly describe keywords, please?

Comment (Fashander) [reply website/docs/contributing/writing-guide.md:80]: Thanks. I'll add it.

**Review by AlekSi — COMMENTED** [review]
"Conform PR" check fails

**Comment by AlekSi** [comment]
@Fashander Please try now

File: website/blog/2022-05-16-using-cla-assistant-with-ferretdb.md [hunk 2]
```diff
@@ -5,6 +5,7 @@ author: Alexey Palazhchenko
 description: Like many other open-source projects, FerretDB requires all contributors to sign [our Contributor License Agreement (CLA)](https://gist.github.com/ferretdb-bot/554e6a30bfcc1d954f3853b4aad95281) to protect them from liability.
 image: /img/blog/cla3.jpg
 date: 2022-05-16
+tags: [compatible applications, open-source]
```
Comment (AlekSi) [inline website/blog/2022-05-16-using-cla-assistant-with-ferretdb.md:8]: `open-source` is not in the list of allowed tags

File: website/blog/2022-05-16-using-cla-assistant-with-ferretdb.md [hunk 3]
```diff
@@ -5,6 +5,7 @@ author: Alexey Palazhchenko
 description: Like many other open-source projects, FerretDB requires all contributors to sign [our Contributor License Agreement (CLA)](https://gist.github.com/ferretdb-bot/554e6a30bfcc1d954f3853b4aad95281) to protect them from liability.
 image: /img/blog/cla3.jpg
 date: 2022-05-16
```
Comment (AlekSi) [inline website/blog/2022-05-16-using-cla-assistant-with-ferretdb.md:7]: I think we could remove all `date:` fields now

File: website/docs/contributing/writing-guide.md [hunk 4]
```diff
@@ -77,6 +77,46 @@ Rather than use relative paths, we strongly suggest the following approach, sinc
...
+
+Tags are an important part of every blog post, and they appear at the top of the front matter.
+They make it easy for readers to search and identify related blog posts based on their categories or subject matter.
+You can view all [currently listed tags here](https://blog.ferretdb.io/tags/).
+
+Please note that tags are case-sensitive, such that `Release` and `release` are two separate tags.
+Unless distinctly written (as in the case with CI/CD, DevOps), all tags should be in small-case.
```
Comment (AlekSi) [inline website/docs/contributing/writing-guide.md:87]: Well, I would say "MongoDB" and "GUI" is in the same group as "DevOps", yet we say "mongodb gui" below. Let's just remove that "unless" part

File: website/docs/main.md [hunk 5]
```diff
@@ -1,6 +1,7 @@
 ---
 sidebar_position: 1
 slug: /
+description: FerretDB Documentation| FerretDB is an open-source MongoDB alternative that translates MongoDB wire protocol queries to SQL, with PostgreSQL as the database engine.
```
Comment (AlekSi) [inline website/docs/main.md:4]: Why do we need "|" and why is there no space before it?

File: website/docs/understanding-ferretdb.md [hunk 6]
```diff
@@ -4,20 +4,54 @@ sidebar_position: 2
 
 # Understanding FerretDB
 
-FerretDB is a document database that uses similar commands, drivers, and tools to those of MongoDB for storing data.
-Unlike relational databases, which use tables, rows, and columns to define the schema of the database, document databases use key-value pairs to build the structure of a document.
+FerretDB is an open source MongoDB alternative that converts MongoDB wire protocols to SQL, with a backend on PostgreSQL.
+As a document database, FerretDB uses similar commands, drivers, and tools to those of MongoDB.
```
Comment (AlekSi) [inline website/docs/understanding-ferretdb.md:8]: I think "similar" is not the right word there: we don't use "similar" drivers – we use MongoDB drivers as-is

File: website/docs/understanding-ferretdb.md [hunk 7]
```diff
@@ -4,20 +4,56 @@ sidebar_position: 2
 
 # Understanding FerretDB
 
-FerretDB is a document database that uses similar commands, drivers, and tools to those of MongoDB for storing data.
-Unlike relational databases, which use tables, rows, and columns to define the schema of the database, document databases use key-value pairs to build the structure of a document.
+FerretDB is an open source MongoDB alternative that converts MongoDB wire protocols to SQL, with a backend on PostgreSQL.
```
Comment (AlekSi) [inline website/docs/understanding-ferretdb.md:7]: @Fashander @ptrfarkas do we want to mention SQLite there?

File: website/docs/understanding-ferretdb.md [hunk 8]
```diff
@@ -4,20 +4,56 @@ sidebar_position: 2
...
-:::note
-Get more information on the key differences [here](diff.md).
+:::tip
+New to FerretDB?
+
+Check out our:
+
+- [Installation guide](/quickstart-guide/)
+- [Key differences](/diff/)
+- [Basic CRUD operations](/basic-operations/)
```
Comment (AlekSi) [inline website/docs/understanding-ferretdb.md:23]: That contradicts out guidelines for links

File: website/docs/understanding-ferretdb.md (hunk 7 above)
Comment (AlekSi) [reply website/docs/understanding-ferretdb.md:7]: Ok, we are not doing that, at least until SQLite support is stable

**Review by AlekSi — APPROVED** [review]

**Pull request merged**